react/nextJsQuestions.md

```diff
@@ -0,0 +1,37 @@
+# Next.js Interview Questions
+
+## General Questions:
+
+[What is Next.js, and how does it differ from traditional React applications?](https://nextjs.org/docs/getting-started/introduction)
+
+[Explain the concept of server-side rendering (SSR) in Next.js](https://nextjs.org/docs/basic-features/data-fetching)
+
+[How does Next.js handle client-side routing?](https://nextjs.org/docs/navigation/introduction)
+
+[What are the benefits of using Next.js for building React applications?](https://nextjs.org/learn/basics/why-use-nextjs)
+
+[What is the purpose of the `getInitialProps` function in Next.js? How is it different from using `getStaticProps` or `getServerSideProps`?](https://nextjs.org/docs/api-reference/data-fetching/getInitialProps)
+
+[How does Next.js handle automatic code splitting, and why is it important?](https://nextjs.org/docs/advanced-features/automatic-code-splitting)
+
+[What is the purpose of the `Link` component in Next.js, and how does it differ from traditional anchor (`<a>`) tags?](https://nextjs.org/docs/api-reference/next/link)
+
+[How can you configure routing in a Next.js application?](https://nextjs.org/docs/basic-features/pages)
```

Comment:
Replace to https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts